src/main/scala/uk/gov/hmrc/ct/ct600/v3/calculations/LoansToParticipatorsCalculator.scala

```diff
@@ -149,4 +168,47 @@ trait LoansToParticipatorsCalculator extends CtTypeConverters {
     case _ => B485(false)
   }
 
+  private def validRepaymentsWihin9Months(loans: List[Loan], date: LocalDate): List[Repayment] = for {
```

Comment:
was undecided whether to move these methods in Loans2Participators itself. What do you think?